MaterialLib/SolidModels/MechanicsBase.h

```diff
@@ -135,6 +144,22 @@ struct MechanicsBase
         return {};
     }
 
+    /// Gets the type of constitutive model
+    virtual ConstitutiveModel getConstitutiveModel() const
+    {
+        return ConstitutiveModel::Invalid;
+    }
+
+    /// Get temperature related coefficient for the global assembly if there is
```

Comment:
does this relate to the Arrhenius contribution? If so, I'd name it that. "Temperature related coefficient" is a little vague ... maybe "ArrheniusTemperatureCoefficient"

Reply:
For me "Arrhenius" always translates to `exp(-E / RT)`. Maybe "Arrhenius" too narrow.

Reply:
I am considering that there may be other temperature models, e.g. temperature dependent cam-clay like model, which has another type of temperature term. Therefore, I would like to have a general name.